tests/foreman/cli/test_activationkey.py

```diff
@@ -874,14 +874,13 @@ def test_positive_delete_subscription(function_entitlement_manifest_org, module_
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
 @pytest.mark.upgrade
-def test_positive_update_aks_to_chost(module_org, module_location, rhel7_contenthost, target_sat):
-    """Check if multiple Activation keys can be attached to a
-    Content host
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_update_aks_to_chost(module_org, module_location, rhel_contenthost, target_sat):
```

Comment:
```suggestion
def test_positive_update_aks_to_chost(module_org, module_location, rhel_contenthost, module_target_sat):
```
To be consistent, I'd suggest using `module_target_sat` in the test.